resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -131,8 +131,14 @@ cloudformation.missing_property=Property {} not found on {}
 cloudformation.yaml.too_many_documents=There should only be 1 YAML document per file
 cloudformation.yaml.invalid_root_type=Template does not start with a mapping
 cloudformation.template_index.missing_type=Resource type must not be null for indexing
+cloudformation.create_stack.failed=Failed to crate stack {0} in status {1}
+cloudformation.create_stack.timeout=Failed to create stack {0} in {1}s. Please go to AWS Console for the latest status of the stack.
+cloudformation.create_stack.failed_validation=Failed to create stack {0} due to validation error
+cloudformation.update_stack.failed=Failed to update stack {0} in status {1}
```

Comment:
Similar: `Failed to update stack {0}: {1}`
I guess alternatively `Failed to update. Stack {0} has status {1}`

Reply:
comment applies to delete below too